tools/mo/openvino/tools/mo/moc_frontend/extractor.py

```diff
@@ -132,9 +132,10 @@ def get_port(match, match_starts_with_name, input_model, framework):
     if len(found_places) == 0:
         raise_no_node(node_name)
 
-    # Check that there is no collision, all found places shall point to same data
-    if not all([n.is_equal_data(found_places[0]) for n in found_places]):
-        raise_node_name_collision(node_name, found_place_names)
+    if framework != "pytorch":
+        # Check that there is no collision, all found places shall point to same data
+        if not all([n.is_equal_data(found_places[0]) for n in found_places]):
+            raise_node_name_collision(node_name, found_place_names)
```

Comment:
why is it so for PT - one tensor name for several tensor places?

Reply:
the first input is always self, it does not have shape and type and invisible for users, it is removed from model after conversion.

Reply:
moved to frontend side